gatsby-plugin-material-ui/package.json

```diff
@@ -28,18 +28,18 @@
     "prepare": "cross-env NODE_ENV=production npm run build"
   },
   "dependencies": {
-    "@emotion/cache": "^11.4.0",
-    "@emotion/server": "^11.4.0"
+    "@emotion/cache": "^11.10.5",
+    "@emotion/server": "^11.10.0"
   },
   "peerDependencies": {
-    "@emotion/react": "^11.4.0",
+    "@emotion/react": "^11.10.5",
     "gatsby": "^3.0.0 || ^4.0.0"
```

Comment:
Is v5 meant to be added here?

Reply:
Yep... That was the first change I made before I had to jump through hoops to get the tests to run (and apparently reverted that bit of the change...)